random_ex.py

```diff
@@ -1,15 +1,34 @@
-#-----------------------------------------------------------------------
+# -----------------------------------------------------------------------
 #
 #   random_ex.py - a python script showing example of random number
 #                  generator
 #
-#---------------------    module history   -----------------------------
+# ---------------------    module history   -----------------------------
 #
 #   12-OCT-18 : PTR XXXX, Created, Dodd M. Thomas
 #
-#-----------------------------------------------------------------------
- 
- 
+# -----------------------------------------------------------------------
+
+
 import random
+print('from random for loop:')
 for i in range(5):
-   print(random.randint(1,100))
```

Comment:
In python, the convention is to have 4 spaces as a "tab". Sometimes your code comes through as 3 spaces.